platform/viewer/src/connectedComponents/ToolContextMenu.js

```diff
@@ -17,232 +14,125 @@ const toolTypes = [
   'RectangleRoi',
 ];
 
-let defaultDropdownItems = [
-  {
-    actionType: 'Delete',
-    action: ({ nearbyToolData, eventData }) => {
-      const element = eventData.element;
-
-      cornerstoneTools.removeToolState(
-        element,
-        nearbyToolData.toolType,
-        nearbyToolData.tool
-      );
-      cornerstone.updateImage(element);
+const ToolContextMenu = ({
+  onSetLabel,
+  onSetDescription,
+  isTouchEvent,
+  eventData,
+  onClose,
+}) => {
+  const mainElement = useRef();
```

Comment:
I feel like there's a better way to dynamically update position?

Reply:
mainElement is not necessary anymore, i can take it out.